manifests/init.pp

```diff
@@ -302,6 +302,10 @@
 #   Default: undef
 #   Hash of 'unicorn' config parameters.
 #
+# [*puma*]
```

Comment:
we really need to convert this to puppet-strings :sadface:

Reply:
https://github.com/voxpupuli/puppet-gitlab/issues/339 ;)